src/test/java/spark/InitExceptionHandlerTest.java

```diff
@@ -16,18 +16,16 @@
 
     @BeforeClass
     public static void setUpClass() throws Exception {
-
         service1 = ignite();
         service1.port(1122);
         service1.init();
         service1.awaitInitialization();
 
         service2 = ignite();
-        service2.port(1122);
+        service2.port(70000);
```

Comment:
Not related to change? I suppose it's a better way of testing it than to fire up two instances though, I will update the test.